Production/crab/configs/Phase2_110X/MuonGun.txt

```diff
@@ -0,0 +1,33 @@
+Automatic
+globalTag=110X_mcRun4_realistic_v3 sampleType=MC_Phase2_110X
+
+Muminus_Pt10-gun_NoPU    1000 /Muminus_Pt10-gun/Phase2HLTTDRWinter20RECOMiniAOD-NoPU_110X_mcRun4_realistic_v3-v2/MINIAODSIM
+Muminus_Pt10-gun_PU140   1000 /Muminus_Pt10-gun/Phase2HLTTDRWinter20RECOMiniAOD-PU140_110X_mcRun4_realistic_v3-v2/MINIAODSIM
+Muminus_Pt10-gun_PU200   1000 /Muminus_Pt10-gun/Phase2HLTTDRWinter20RECOMiniAOD-PU200_110X_mcRun4_realistic_v3-v2/MINIAODSIM
```

Comment:
I don't think that muon gun with pt = 10 GeV will be useful for the training, because the cut on tau candidates stored in miniAOD is 18 GeV.

Reply:
Ok. Then I'll remove these samples from the list.

Reply:
Done